atomic

```diff
@@ -501,6 +518,23 @@ if __name__ == '__main__':
                           help=_("filesystem location of image/container to "
                                  "be unmounted"))
 
+    # atomic update-container
+    updatecontainerp = subparser.add_parser(
+        "update-container", help=_("update a container"),
+        epilog="update a container.")
+    updatecontainerp.set_defaults(func='update_container')
```

Comment:
How is this different then update? Can we just improve atomic update rather then add this command?